src/Symfony/Component/Serializer/Normalizer/ScalarDenormalizer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\PropertyInfo\Type;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+
+final class ScalarDenormalizer implements DenormalizerInterface, CacheableSupportsMethodInterface
```

Comment:
Maybe register the new denormalizer in `src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.php`?

Reply:
Done